go/libraries/doltcore/env/remotes.go

```diff
@@ -133,17 +131,20 @@ func (r *Remote) GetRemoteDBWithoutCaching(ctx context.Context, nbf *types.NomsB
 	return doltdb.LoadDoltDBWithParams(ctx, nbf, r.Url, filesys2.LocalFS, params)
 }
 
-type PushMeta struct {
-	Opts   []*PushOpts
+// PushOptions contains information needed for push for
+// one or more branches or a tag for a specific remote database.
+type PushOptions struct {
+	Opts   []*PushTarget
```

Comment:
Mind changing `Opts` to `Targets`? That'll read a little better. Looks like there are only a couple of references to `Opts` so should be easy to change.